cli_reference/developer_cli_odo/understanding-odo.adoc

```diff
@@ -40,46 +40,51 @@ In `{odo-title}`, services are provisioned from the OpenShift Service Catalog an
 [id="odo-supported-languages-and-images"]
 === Officially supported languages and corresponding container images
 
-.Supported languages, container images, and package managers
+.Supported languages, container images, package managers, and platforms
 [options="header"]
 |===
-|Language |Container image |Package manager
+|Language |Container image |Package manager |Platform
 |*Node.js*
-|https://registry.centos.org/centos/nodejs-10-centos7/latest[centos/nodejs-10-centos7]
-|NPM
-
-|
-|https://hub.docker.com/r/centos/nodejs-12-centos7[centos/nodejs-12-centos7]
+|https://access.redhat.com/containers/#/registry.access.redhat.com/rhscl/nodejs-10-rhel7[rhscl/nodejs-10-rhel7]
 |NPM
+|amd64, s390x
 
 |
 |https://catalog.redhat.com/software/containers/rhscl/nodejs-12-rhel7/[rhscl/nodejs-12-rhel7]
 |NPM
+|amd64, s390x
 
 |
-|https://www.github.com/bucharest-gold/centos7-s2i-nodejs[bucharestgold/centos7-s2i-nodejs]
+|https://access.redhat.com/articles/3376841[rhoar-nodejs/nodejs-10]
 |NPM
+|amd64
 
 |
-|https://access.redhat.com/containers/#/registry.access.redhat.com/rhscl/nodejs-8-rhel7[rhscl/nodejs-8-rhel7]
+|https://registry.centos.org/centos/nodejs-10-centos7/latest[centos/nodejs-10-centos7]
```

Comment:
This link doesn't work either, should it?

Reply:
Sorry I didn't think about validating the existing links. I got two now that work and will replace.

Reply:
Thank you!

Reply:
Will you use "https://hub.docker.com/r/centos/nodejs-10-centos7" to replace  "https://registry.centos.org/centos/nodejs-10-centos7/latest"

Reply:
For nodejs-10-centos7 , the upstream doc uses this link: https://github.com/sclorg/s2i-nodejs-container
For nodejs-12-centos7 , the upstream doc uses this link: https://github.com/sclorg/s2i-nodejs-container
@SNiemann15 Hope it will help you.

Reply:
@boczkowska said she will update the urls. Yana will you or shall I give it another try and see if my "git" problems still persist?